doc/user/user_commands/ksu.rst

```diff
@@ -12,7 +12,8 @@ SYNOPSIS
 [ **-c** *source_cache_name* ]
 [ **-k** ]
 [ **-r** time ]
-[ **-pf** ]
```

Comment:
`-pf` is still listed on line 250.  (I'd put this comment there, but github doesn't seem to let me do that anymore.)

Reply:
Fixed.  I also changed the usage message in the C code not to say ``-pfPF``, since there's room to say ``[-p | -P] [-f | -F]``.